compiler/rustc_parse/src/validate_attr.rs

```diff
@@ -194,6 +194,11 @@ pub fn check_attribute_safety(psess: &ParseSess, safety: AttributeSafety, attr:
             }
         }
     } else if let Safety::Unsafe(unsafe_span) = attr_item.unsafety {
+        // Allow (but don't require) `#[unsafe(naked)]` so that compiler-builtins can upgrade to it.
```

Comment:
Could this be a `FIXME`?

Reply:
sure, I've made it link back to this PR, which should provide sufficient context. 

Also, Ideally this should only be there for a couple of days, the PRs to remove it again are already more or less done, so it's just a matter of getting this change into a nightly build, updating compiler-builtins, and then making `#[naked]` a proper unsafe attribute.